gulp/tasks/images.js

```diff
@@ -0,0 +1,53 @@
+import webp from "gulp-webp";
+import imagemin from "gulp-imagemin";
+
```

Comment:
very strange pipelines, if you do several things in parallel, devide this pipeline into several tasks and perform them in parallel

Reply:
Done

Reply:
it doesn't work

Reply:
What doesn't work?